trex-scripts/tests/port_shaping_rate.py

```diff
@@ -58,6 +56,11 @@ def start(self, args: dict) -> None:
 
         # Get statistics for TX and RX ports
         stats = self.client.get_stats()
+        rx_rate = stats[1]['rx_bps'] / (10**6)
```

Comment:
Same comment here about adding unit to the variable name
```suggestion
        rx_rate_mbps = stats[1]['rx_bps'] / (10**6)
```